doc/user.pod

```diff
@@ -3072,8 +3072,8 @@ created using the following functions.
 		__isl_take isl_set *set,
 		__isl_take isl_qpolynomial *qp);
 
-The following convenience functions first create a base expression and
-then create a piecewise expression over a universe domain.
+The following convenience functions first create a base expression and then
+create a piecewise expression over a universe domain of a given (local) space.
```

Comment:
I'm not sure that "over a universe domain of a space" is mathematically correct.  "over domain that is a universe set in the given (local) space", leaving that to @skimo-openhub.

Reply:
Done.